build/cmake/libmongoc-1.0-config.cmake.in

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017 MongoDB Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if(NOT libmongoc-1.0_FIND_QUIETLY)
+    message(WARNING "This CMake package is deprecated. Prefer instead to use the \"mongoc-1.0\" package and link to mongo::mongoc_shared.")
+endif()
+
+set (MONGOC_MAJOR_VERSION @libmongoc_VERSION_MAJOR@)
+set (MONGOC_MINOR_VERSION @libmongoc_VERSION_MINOR@)
+set (MONGOC_MICRO_VERSION @libmongoc_VERSION_PATCH@)
+set (MONGOC_VERSION @libmongoc_VERSION@)
+set (MONGOC_VERSION_FULL @libmongoc_VERSION_FULL@)
+
+include(CMakeFindDependencyMacro)
+find_dependency (mongoc-1.0 "@libmongoc_VERSION_MAJOR@.@libmongoc_VERSION_MINOR@")
```

Comment:
Is `@libmongoc_VERSION_MAJOR@.@libmongoc_VERSION_MINOR@` needed? libbson-1.0-config.cmake.in does does not include the version in the call to `find_dependency`.